code/datums/gamemode/factions/syndicate/nukeops.dm

```diff
@@ -203,4 +203,19 @@
 
 	else
 		nuke_code = "code will be provided later"
-	return
\ No newline at end of file
+	return
+
+/datum/faction/syndicate/nuke_op/process()
+	var/livingmembers
+	var/mob/living/M
+	if(members.len > 0)
+		for (var/datum/role/R in members)
+			if(R.antag.current)
+				M = R.antag.current
+				if(M.stat != DEAD)
+					livingmembers++
+		if(livingmembers > 0)
```

Comment:
this  and the next line can be expressed more simply as `if(!livingmembers && stage < FACTION_DEFEATED)` to avoid the extra return and else

I should note that this might be slightly non-atomic as a change though, because setting FACTION_DEFEATED will attempt to summon a shuttle if one is not already on the way.  Same with setting FACTION_ENDGAME which automatically upgrades to red alert.

Reply:
Would it be better for nukeops to internally track their own "endgame", or should a separate change move the shuttle call/alert status behavior to its own faction flag so that stage() can be used more generically?

Reply:
It doesn't make sense to implement it like that at all. The only behavior you want from endgame is that it plays music, you don't want any other functions from staging. So just call StartThematic and StopThematic directly.

Reply:
I think it makes some amount of sense to do it since it would allow other/new factions to use the faction stages however they want but I can also see why it shouldn't be changed. 

Either way, the button doesn't use stages to track progress now.